rfcs/055-prismic-api/events.md

```diff
@@ -0,0 +1,108 @@
+# Prismic API: Events endpoint
+
+## Example response
+
+```
+GET /events/{id}
+{
+    "type": "Event",
+    "id": "Y8aBHhEAAKRE_9AF",
+    "format": {
+        "id": "Wd-QYCcAACcAoiJS",
+        "label": "Discussion"
+        "type": "EventFormat"
+    },
+    "title": "Our Microbes and Our Health",
+    "isOnline": false,
+    "availableOnline": true,
+    "isRelaxed": false,
+    "caption": "Join Claudia Hammond from the BBC and a panel of experts to consider the explosion of scientific knowledge about the millions of microbes that live in and on our bodies. ",
+    "interpretations": [
+        {
+            "id": "WmXl4iQAACUAnyDr",
+            "label": "Captioned (screen)",
+            "type": "EventInterpretation"
+        },
+        ...
+    ],
+    "audiences": [
+        {
+            "id": "WlYWByQAACQAWdA0",
+            "label": "Schools",
+            "type": "EventAudience"
+        },
+        ...
+    ],
+    "instantiations": [
+        {
+            "start": "2018-12-20T10:44:15+0000",
+            "end": "2018-12-20T16:44:15+0000",
+            "place": {
+                "id": "Wn1fvyoAACgAH_yG",
+                "label": "Reading Room",
+                "type": "Place"
+            },
+        }
+    ],
+    "contributors": [ // behind ?includes=contributors
+        {
+            ... // reuse the Contributor model (without primaryContributor)
```

Comment:
Are we sure about that? I can see arguments either way, but I'm not sure it's where I would start.

Reply:
[see the comment above](https://github.com/wellcomecollection/docs/pull/90#discussion_r1121561757)